t/api/04-jobs.t

```diff
@@ -222,15 +222,19 @@ subtest 'job overview' => sub {
 
 $schema->txn_begin;
 
-subtest 'restart jobs' => sub {
-    $t->post_ok('/api/v1/jobs/restart', form => {jobs => [99981, 99963, 99962, 99946, 99945, 99927, 99939]})
-      ->status_is(200);
+subtest 'restart jobs, error handling' => sub {
+    note 'cluster for 99962';
+    use Data::Dumper;
+    note Dumper($schema->resultset('Jobs')->find(99962)->cluster_jobs);
```

Comment:
I don't think you need that, explain from Test::Most should be enough, no dumper needed

Reply:
That's just a leftover from debugging. I'll remove it.